deploy/cr.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: cluster1
 #  finalizers:
 #  - percona.com/delete-pvc
+#  - percona.com/delete-tls
```

Comment:
In our other operators this is named `delete-ssl`, we should make it the same here as well?

Reply:
you're right, switched to `delete-ssl`